src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -925,12 +939,20 @@ void BLEManagerImpl::HandleTxConfirmationEvent(BLE_CONNECTION_OBJECT conId)
     event.Type                          = DeviceEventType::kCHIPoBLEIndicateConfirm;
     event.CHIPoBLEIndicateConfirm.ConId = conId;
     PlatformMgr().PostEventOrDie(&event);
+    CancelBleSendIndicationTimeoutTimer();
 }
 
-// TODO:: Need to Implement
+
 void BLEManagerImpl::HandleSoftTimerEvent(void)
 {
-    // TODO:: Need to Implement
+    uint8_t connHandle = 1;
+    ChipLogProgress(DeviceLayer, "BLEManagerImpl::HandleSoftTimerEvent CHIPOBLE_PROTOCOL_ABORT");
```

Comment:
what is this log ?

Reply:
Log to tell we are aborting the connection because of timeout

Reply:
Is this an error case? The function name suggests a handler for Soft Timer Event.

Reply:
Yes it is same in EFR32 as well